src/libsync/account.h

```diff
@@ -125,6 +125,8 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject
      */
     QString davPath() const;
 
+    static QString davPathBase();
```

Comment:
would it make sense to make that one private or even put into the `.cpp` file? I don't think that needs to be part of the public api.

Reply:
@FlexW No. It is used in cmd.cpp :(

Reply:
@FlexW Made it private.